src/modules/packages/main.py

```diff
@@ -464,6 +524,10 @@ def install(self, pkgs, from_local=False):
     def remove(self, pkgs):
         self.reset_progress()
         self.run_pacman(["pacman", "-Rs", "--noconfirm"] + pkgs, True)
+        
+    def remove_deps(self, pkgs):
+        self.reset_progress()
+        self.run_pacman(["pacman", "-Rcns", "--noconfirm"] + pkgs, True)
```

Comment:
This should probably be `"-Rc"`.  We don't use `n` in the other options so we probably shouldn't do it here.  Combining `-Rs` and -`Rc` is not a good idea in an installer.  It removes packages recursively in both directions and nearly always takes more packages away than it should.  You would commonly have situations where you are removing packages the user selected to install in prior steps.  `-Rc` is sufficient to ensure your package gets removed.

Reply:
Fixed